templates/components/products/bulk-discount-rates.html

```diff
@@ -1,12 +1,12 @@
 {{#if bulk_discount_rates.length}}
     {{#if product}}<dt class="productView-info-name">{{lang 'products.bulk_pricing.title'}}</dt>{{/if}}
     <dd class="productView-info-value">
-        <a href="{{url}}"
-           {{#unless is_ajax }}data-reveal-id="bulkPricingModal{{#if id}}-{{id}}-{{$index}}{{/if}}"{{/unless}}>
+        <a href="{{product.url}}#bulk_pricing"
```

Comment:
This makes sure that the product URL is in the link, though it hasn't caused issues before. Follows the same convention as `#write_review`